### PR TITLE
Add evergreen VPN articles [fix #10224]

### LINKED ISSUE #10224: Migrate VPN blog posts to www
We want to redirect the evergreen vpn blog posts to /products/vpn/ so they rank better long term.

****

This can use a layout similar to:
https://www.mozilla.org/en-US/firefox/features/block-fingerprinting/

We should design a template that will have the VPN Look & Feel

**** 

Brief:

https://docs.google.com/document/d/1PDVv1I-jkTFvRX-jYDtPhgn9Zhq16EUUZS5EH7hJjjM/edit?usp=sharing

****
Success Criteria
- [x] Design template to use for all pages
- [x] Code template to use for all pages
- [x] Code pages
- [x] Link to FAQ on homepage
- [ ] Launch pages (Date to be provided by Rebecca)
- [ ] Redirect the blog posts to www when they are online

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Adds four new VPN-related content pages, migrated from previous blog posts. Once they're live we can redirect the original URLs from the blog to these pages.

## Issue / Bugzilla link
#10224 

## Testing
http://localhost:8000/products/vpn/#faq
http://localhost:8000/products/vpn/more/what-is-an-ip-address/
http://localhost:8000/products/vpn/more/what-is-a-vpn/
http://localhost:8000/products/vpn/more/when-to-use-a-vpn/
http://localhost:8000/products/vpn/more/vpn-or-proxy/

### CODE REVIEW

**Comment by craigcook** [comment]
Marking DNM for the moment until we get signoff.

**Comment by craigcook** [comment]
We were holding these pages back because another team was doing a newsletter test link to some of the original blog posts and we wanted to wait for that to complete before redirecting the URLs. According to Rebecca that should be done next week and this will be unblocked.

So it's ready for code review before that and we just shouldn't merge/push until we're all clear.

File: l10n/en/products/vpn/more/ip-address.ftl
Comment (alexgibson) [inline l10n/en/products/vpn/more/ip-address.ftl:11]: The end of this string has two periods
```suggestion
vpn-ip-address-mozilla-answers-all-of = { -brand-name-mozilla } answers all of your questions about IP addresses from what is an IP address, to how it works, and if you should hide it.
```

File: l10n/en/products/vpn/more/ip-address.ftl
Comment (alexgibson) [inline l10n/en/products/vpn/more/ip-address.ftl:51]: The brand name here should be `Mozilla VPN`?
```suggestion
vpn-ip-address-a-vpn-is = A VPN is a way to hide your IP address. <a href="{ $vpn }">When you use a VPN</a>, your external IP address will be coming from the VPN server’s external IP, rather than your location’s external IP address. So if your connecting VPN server is located in California, your external IP will look like it’s connected from California, no matter where you actually are. Plus, your online activity is sent over an encrypted, secure connection to your VPN server, giving you additional security and privacy. <a href="{ $mozvpn }">{ -brand-name-mozilla-vpn }</a> is one way to hide your IP address. We don’t keep activity logs or partner with third-party analytics platforms. We offer full-device protection for up to five devices with servers in 30+ countries, you can connect to anywhere, from anywhere.
```

File: l10n/en/products/vpn/more/vpn-or-proxy.ftl
Comment (alexgibson) [inline l10n/en/products/vpn/more/vpn-or-proxy.ftl:20]: Should `enhanced tracking protection` be capitalized since it's a product feature?
```suggestion
vpn-or-proxy-when-you-use-firefox = When you <a href="{ $firefox }">use { -brand-name-firefox }</a>, <a href="{ $etp }">Enhanced Tracking Protection</a> automatically blocks many third party <a href="{ $trackers }">web trackers</a> from following you around the web. But here’s an interesting fact: your internet service provider (ISP) that you are paying for an internet connection — can still observe and track you.
```

File: l10n/en/products/vpn/more/vpn-or-proxy.ftl
Comment (alexgibson) [inline l10n/en/products/vpn/more/vpn-or-proxy.ftl:36]: Extra space between `your  traffic`
```suggestion
vpn-or-proxy-vpns-do-more = VPNs do more than proxies in that a proxy only protects what you do in your browser, whereas a VPN protects all your traffic, including your browser, wherever you have a VPN installed and enabled. VPNs provide added security and privacy for all your online activity—an important consideration if you want to keep your activity to yourself and make it more difficult for data hungry trackers and ISPs to create a profile of you across all your devices, like your phone, computers and tablet.
```

File: l10n/en/products/vpn/more/vpn-or-proxy.ftl
Comment (alexgibson) [inline l10n/en/products/vpn/more/vpn-or-proxy.ftl:43]: Should 30 be a variable here? https://github.com/mozilla/bedrock/blob/master/bedrock/settings/base.py#L1569

File: l10n/en/products/vpn/more/what-is-a-vpn.ftl
Comment (alexgibson) [inline l10n/en/products/vpn/more/what-is-a-vpn.ftl:23]: Make 30 a variable?

File: media/static-bundles.json
Comment (alexgibson) [inline media/static-bundles.json:1468]: Are these needed? The pages just seem to link to the landing page so may not be necessary.

File: bedrock/products/templates/products/vpn/more/base-article.html
Comment (alexgibson) [inline bedrock/products/templates/products/vpn/more/base-article.html:43]: Nit: indentation

File: l10n/en/products/vpn/more/what-is-a-vpn.ftl
Comment (alexgibson) [inline l10n/en/products/vpn/more/what-is-a-vpn.ftl:13]: Should `virtual private network (VPN)` be capitalized?
```suggestion
vpn-what-is-many-of-us = Many of us spend multiple hours a day using the internet to do everyday things like watching videos, shopping, gaming and paying bills, all the way to managing complex work projects and having confidential video calls. A <strong>Virtual Private Network (VPN)</strong> is one of the best ways to stay private and secure online, and keep your personal data protected.
```

File: bedrock/products/templates/products/vpn/more/ip-address.html
Comment (alexgibson) [inline bedrock/products/templates/products/vpn/more/ip-address.html:39]: May or may not be important - should owned blog links contain utm params to track referrals?

File: bedrock/products/templates/products/vpn/more/base-article.html
Comment (alexgibson) [inline bedrock/products/templates/products/vpn/more/base-article.html:51]: Use brand name?

```suggestion
      <h2 class="mzp-c-wordmark mzp-t-wordmark-md mzp-t-product-vpn">{{ ftl('vpn-shared-product-name') }}</h2>
```

File: bedrock/products/templates/products/vpn/more/vpn-or-proxy.html
Comment (alexgibson) [inline bedrock/products/templates/products/vpn/more/vpn-or-proxy.html:22]: Not entirely sure how you might handle this, but this image feels like it's more than decorative and could maybe use some `alt` text?

image

The difference is explained in the paragrahps below, but the image does serve as a nice quick summary.

File: bedrock/products/templates/products/vpn/more/vpn-or-proxy.html
Comment (alexgibson) [inline bedrock/products/templates/products/vpn/more/vpn-or-proxy.html:28]: UTM params on FPN referral links?

File: bedrock/products/templates/products/vpn/more/base-article.html
Comment (alexgibson) [inline bedrock/products/templates/products/vpn/more/base-article.html:33]: From a user perspective It feels a little odd that the button reads "Get Mozilla VPN", and when I click it I'm taken to an almost identical looking page with another button that reads "Get Mozilla VPN". I have to click that again and it then scrolls down to the pricing.

Would it be worth considering linking to `{{ url('products.vpn.landing') }}#pricing` to eliminate the second click?

**Review by alexgibson — CHANGES_REQUESTED** [review]
Pages look good! Mostly small nits and suggestions, no blockers

File: bedrock/products/templates/products/vpn/more/vpn-or-proxy.html
Comment (craigcook) [reply bedrock/products/templates/products/vpn/more/vpn-or-proxy.html:22]: I thought about it, but since the text does explain everything I decided the illustration was supplemental. Trying to describe it adequately in a text alternative would probably have to repeat much of the same info in a condensed summary, and not actually describe the image. So I think it's probably OK to leave alt blank and treat this image as optional.

File: l10n/en/products/vpn/more/vpn-or-proxy.ftl
Comment (craigcook) [reply l10n/en/products/vpn/more/vpn-or-proxy.ftl]: Good call. I did the same on the What Is a VPN page as well.

**Comment by craigcook** [comment]
Thanks for the thorough review! Updated and squashed.

**Review by alexgibson — APPROVED** [review]

**Comment by alexgibson** [comment]
@craigcook only noticed after I merged sorry. Looks like we forgot to add a config for the fluent files? https://github.com/mozilla/bedrock/blob/master/l10n/configs/pontoon.toml#L278-L286

**Comment by craigcook** [comment]
> @craigcook only noticed after I merged sorry. Looks like we forgot to add a config for the fluent files? https://github.com/mozilla/bedrock/blob/master/l10n/configs/pontoon.toml#L278-L286

Oops! I'll add it in another PR now. Sorry bout that.

**Pull request merged**
